trivup/trivup.py

```diff
@@ -642,5 +645,72 @@ def __str__(self):
                                    self.appid, self.state)
 
 
+class DockerApp(App):
+    __slots__ = ['image', 'container_name', 'docker_args', "app_args"]
```

Comment:
inconsistent quoting.

what do you need the slots for?

Reply:
Need is a strong word. I just like that it prevents users from adding attributes arbitrarily.